beacon-chain/p2p/pubsub.go

```diff
@@ -147,9 +147,39 @@ func (s *Service) pubsubOptions() []pubsub.Option {
 		pubsub.WithGossipSubParams(pubsubGossipParam()),
 		pubsub.WithRawTracer(gossipTracer{host: s.host}),
 	}
+
+	if len(s.cfg.StaticPeers) > 0 {
+		s.addDirectPeersOption(psOpts)
+	}
+
 	return psOpts
 }
 
+// addDirectPeersOption parses the static peers from the service configuration and adds
+// the direct peers option to the pubsub options.
+func (s *Service) addDirectPeersOption(psOpts []pubsub.Option) {
```

Comment:
You will need to return the opts, as it becomes a different slice below.

Reply:
Sorry for that! I've written very few lines of Go code so far :). I thought `psOpts` was passed by reference. Now I've enforced it and the underlying data should be modified.
Tried to run `golangci-lint` locally and seems good now